src/views/Layout.vue

```diff
@@ -178,14 +178,31 @@ export default {
      */
     this.$root.$on('alert', ({ message, ...params }) => this.toast(message, params))
     this.$root.$on('reminder.show', this.showReminder)
+
+    this.$root.$on('sidebar-show', this.checkNamespaceSidebar)
   },
 
   beforeDestroy () {
     this.$root.$off('alert')
     this.$root.$off('reminder.show', this.showReminder)
+    this.$root.$off('sidebar-show', this.checkNamespaceSidebar)
   },
 
   methods: {
+    checkNamespaceSidebar (handle) {
+      const defaultDisabledRoutes = [
+        'namespaces',
+        'namespace.list',
+        'namespace.edit',
+        'namespace.create',
+        'namespace.clone',
+        'namespace.manage',
+      ]
+      const namespaceRoutes = ['page', 'pages']
```

Comment:
Not sure this is all the routes, try opening a record page when you have sidebar hidden

Reply:
Yeah, totally valid point. Thanks for bringing up.